app/src/main/java/agency/five/codebase/android/movieapp/ui/component/MovieCategory.kt

```diff
@@ -0,0 +1,73 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import androidx.annotation.StringRes
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.IntrinsicSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Divider
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+sealed class MovieCategoryLabelTextViewState
+class MovieCategoryLabelTextViewStateInput(val text: String): MovieCategoryLabelTextViewState()
+class MovieCategoryLabelTextViewStateResource(@StringRes val textRes: Int): MovieCategoryLabelTextViewState()
+
+data class MovieCategoryLabelViewState(
+    val itemId: Int,
+    val isSelected: Boolean,
+    val categoryText: MovieCategoryLabelTextViewState
+)
+
+@Composable
+fun MovieCategoryLabel(
```

Comment:
This component should be clickable and isSelected should toggle when clicked. 
Please pass lambda method: `onItemClick: (MoviewCategoryLabelViewState) -> Unit` which will be called when the component is clicked. In preview pass the correct method which will mutate the state. 
You should have something like moveCategoryState by remember {....} and then mutate it inside the passed lambda. 
When you start the preview on your phone clicking on label should toggle state.